android/rctmgl/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapView.java

```diff
@@ -1258,6 +1258,9 @@ private void updateLocationLayer() {
 
     private WritableMap makeRegionPayload(boolean isAnimated) {
         CameraPosition position = mMap.getCameraPosition();
+        if (position == null || position.target == null) {
+            return new WritableNativeMap();
```

Comment:
I checked and in fact `GeoJSONUtils.toPointFeature()` doesn't support a `null` value for the location, neither does this method. While a `null` check seems the a good next step, two questions come to mind:
1. Do you expect any side issues from returning an empty map here?
2. Could `position` being `null` here indicative of a larger issue, or is it expected to have a `null` value in some circumstances?

Reply:
This empty map is being returned as a payload for MapChangeEvent. At least in Mapbox code itself, I’m not seeing any usage of this event or its payload.

Since this is the only crash log we have, it is not clear exactly at what step this happens. However, I assume onCameraIdle would be called at some step where camera is not active (due to not having location or activity closing?). So it might be possible that it has no location set.

Reply:
Thanks @zugaldia